questions-js1/6.md

```diff
@@ -23,4 +23,8 @@ b) What will `result2` evaluate to? Explain your answer
 c) Try to summarise the main difference between `logSum` and `calculateSum`
 
 
-{YOUR ANSWERS HERE}
\ No newline at end of file
+{YOUR ANSWERS HERE}
+
+a) result1 evaluates the value of a and b, adds those together, and attaches it to a function.
+b) result2 prints the sum of a and b to the web console.
+c) The main difference between calculateSum function and logSum function is, calculateSum only collect the data, does the function and keeps it. But logSum, collect the input, does the function and print it to the console.
```

Comment:
I think you broke down the stages clearly and added sufficient detail to make the explanation clear.